cypress/e2e/tests/migration/applicationinventory/assessment/copy_assessment/copy_assessment.test.ts

```diff
@@ -145,19 +125,13 @@ describe(["@tier2"], "Copy assessment and review tests", () => {
 
         // Select all the applications on page
         clickWithin(modal, "button[aria-label='Select']");
-        if (applicationList.length < 11) {
-            cy.get("ul[role=menu] > li")
-                .contains(`Select page (${applicationList.length} items)`)
-                .click();
-        } else {
-            cy.get("ul[role=menu] > li").contains("Select page (10 items)").click();
-        }
+        cy.get("ul[role=menu] > li").contains("Select page").click();
         cy.get(copy).should("be.visible").should("not.be.disabled");
         clickWithin(modal, "button[aria-label='Select']");
```

Comment:
I think that selecting all apps on page really can be moved to static method in corresponding class

Reply:
@ibragins Now that applications associated with waves cannot be deleted . 
So "Select All" and "Select Page " button is used only in this test . So no need to create these methods . These are only one time use calls .

Reply:
OK